extern/rust/Makefile.mk

```diff
@@ -6,13 +6,13 @@ LIBCORE_DIR=$(BUILD_DIR)/core-$(RUSTC_VERSION)
 
 $(EXTERN_DIR)rust/rustc-$(RUSTC_VERSION)-src.tar.gz:
 	@echo "Need libcore to compile Tock: fetching source $(@F)"
-	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz
+	@wget -q -O $@ https://github.com/rust-lang/rust/archive/$(RUSTC_VERSION).tar.gz --no-check-certificate
```

Comment:
That seems dangerous. Does cygwin not have builtin SSL certificates or something?